### PR TITLE
Use a select for categories on approve instead of a textbox. 

### PR DESCRIPTION
**Pull request by grantg012**
Currently the control for the category on the approve page uses a textbox. This is a little error prone if someone needs to switch the category or if they inadvertently edit it. The PR changes to a select control that only allows the prefilled options.

### CODE REVIEW

**Review by rakoskr6 — COMMENTED** [review]
I haven't run this yet to confirm it's fine but it generally appears fine. I'd suggest adding the php database request (the part with the sql code to get the category list towards the beginning so we can keep all database queries at the top and make the html easier to follow. Also I'm curious why you created a sperate dev file instead of just putting the variable directly in the header?

**Comment by grantg012** [comment]
Regarding the dev file, I wanted something that git wouldn't track. That way the same code could run and produce different results. If anyone ever did a `git checkout -- .` it wouldn't be overwritten.

**Comment by grantg012** [comment]
The database query has been moved

File: approve/index.php [hunk 1]
```diff
@@ -13,36 +13,50 @@
...
+        $items .= $row['purchaseID'];
+        $items .= '">';
+        $items .= $row['item'];
+        $items .= '</option>\n';
+    }
+    //echo $items;
+
+    $categorylist = "";
+    $committee = $_SESSION['committee'];
+    $sql = "SELECT category FROM `Budget` WHERE committee='$committee' AND year='2017-2018'";
```
Comment (grantg012) [inline approve/index.php:42]: Did this for testing because that's what was there. I may end up adding the fiscal year change to this PR

**Comment by grantg012** [comment]
Alright just tested all this, though not too in depth. Lmk what think when you review it @rakoskr6

**Pull request merged**